apps/api/src/domain/models/__tests__/command-helpers/generate-command-fuzz-test-cases.ts

```diff
@@ -10,9 +10,16 @@ type CommandFuzzTestCase = {
 
 export const generateCommandFuzzTestCases = (CommandCtor: Ctor<ICommand>): CommandFuzzTestCase[] =>
     Object.entries(getCoscradDataSchema(CommandCtor)).flatMap(([propertyName, propertySchema]) =>
-        new FuzzGenerator(propertySchema).generateInvalidValues().map(({ value, description }) => ({
-            propertyName,
-            invalidValue: value,
-            description,
-        }))
+        new FuzzGenerator(propertySchema)
+            .generateInvalidValues()
+            .map(({ value, description }) => ({
+                propertyName,
+                invalidValue: value,
+                description,
+            }))
+            .concat({
```

Comment:
Why didn't this cause problems for 'GRANT_RESOURCE_READ_ACCESS_TO_USER'?

Reply:
Because we happened to skip the fuzz testing there. I have added a todo and [this](https://www.pivotaltracker.com/story/show/182840154) story.